sqldelight-gradle/android-plugin/src/test/integration/src/main/java/com/squareup/sqldelight/integration/MyPerson.java

```diff
@@ -2,4 +2,4 @@
 
 import com.google.auto.value.AutoValue;
 
-@AutoValue public abstract class MyPerson extends PersonModel {}
+@AutoValue public abstract class MyPerson implements Person {}
```

Comment:
and it still works anyways /shrug

Reply:
AV works with bean-style naming. We just never used it because it's annoying.

Reply:
yea. You'll still get the property style if you use it from kotlin so thats good enough i think